client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -236,18 +275,29 @@ export default defineComponent({
         @begin-multicam-import="multiCamImport($event)"
         @abort="importMultiCamDialog = false"
       />
-      <v-card
-        v-else-if="checkingMedia"
-        outlined
-      >
-        <v-card-title class="text-h5">
-          Calculating...
-          <v-progress-linear
-            indeterminate
-            color="light-blue"
-          />
-        </v-card-title>
-      </v-card>
+    </v-dialog>
+    <v-dialog
```

Comment:
@BryonLewis I split the current dialog into two separate dialogs, one for import / bulk data import, and the other for import status and the multicam dialog. I tested it locally and believe it works as expected, but I'm not too familiar with multicam stuff.